BUILD.bazel

```diff
@@ -12,8 +12,11 @@ cmake(
         "CMAKE_BUILD_TYPE": "Release",
         "BUILD_SHARED_LIBS": "OFF",
         "BUILD_TESTING": "OFF",
-        "CMAKE_USE_OPENSSL": "FALSE",
         "CURL_DISABLE_LDAP" : "TRUE",
```

Comment:
Did you want to remove this? Should it also be in the `libcurl_boringssl` target? Also, can we please get a comment here why you needed to disable it so someone (possibly you!) in the future can understand why we needed to add it?

Reply:
Found a way to enable LDAP support on Windows! Not disabling it anymore.